src/appleseed.shaders/CMakeLists.txt

```diff
@@ -145,7 +145,6 @@ source_group ("include\\appleseed\\transform" FILES
 
 set (src_blenderseed_sources
      src/blenderseed/as_blender_areaLight.osl
-     src/blenderseed/as_closure2surface.osl
```

Comment:
Is this shader currently used by blenderseed? If yes, we can't simply remove it, that would break all scenes created until the next release of appleseed/blenderseed.

Reply:
We have an exact copy of it in the appleseed folder. In my tests, we pick the appleseed one without issues.

Reply:
Ok, cool.